src/v2/Components/NavBar/NavBarLoggedInActions.tsx

```diff
@@ -52,25 +46,14 @@ export const NavBarLoggedInActions: React.FC<Partial<
         dropdown={<NavBarNotificationsQueryRenderer />}
         placement="bottom-end"
         offset={0}
+        openDropdownByClick
       >
         {({ anchorRef, anchorProps, visible }) => (
           <NavBarItemButton
             ref={anchorRef as any}
             active={visible}
             {...anchorProps}
           >
-            <NavBarItemUnfocusableAnchor
-              href="/works-for-you"
-              onClick={() => {
-                trackEvent({
-                  action_type: AnalyticsSchema.ActionType.Click,
-                  subject: AnalyticsSchema.Subject.NotificationBell,
-                  new_notification_count: getNotificationCount(),
-                  destination_path: "/works-for-you",
-                })
-              }}
-            />
-
```

Comment:
FYI I asked Louis if it would be ok if we no longer track this event and am waiting for a response.